src/IDelegateRegistry.sol

```diff
@@ -10,40 +10,61 @@ pragma solidity ^0.8.20;
 interface IDelegateRegistry {
     /// @notice Delegation type
     enum DelegationType {
-        NONE,
         ALL,
         CONTRACT,
-        ERC721,
         ERC20,
+        ERC721,
```

Comment:
Makes sense from an EIP ordering perspective, but wonder if it makes sense to preserve the same enum ordering from v1 here

Reply:
Happy to go either way. Think we can get probably get away with the EIP ordering since we're changing the name from `TOKEN` in v1 to `ERC721` in v2

Reply:
Reverted to the v1 ordering, made ordering consistent in the interface and registry.